srcpkgs/qemu/template

```diff
@@ -81,6 +81,15 @@ do_install() {
 	make DESTDIR=${DESTDIR} install
 	# qemu-bridge-helper must be setuid for non privileged users.
 	chmod u+s ${DESTDIR}/usr/libexec/qemu-bridge-helper
+	# The firmware JSON files should be searched for in three directories
+	# * /usr/share/qemu/firmware
+	# * /etc/qemu/firmware
+	# * $XDG_CONFIG_HOME/qemu/firmware
+	# See: https://git.qemu.org/?p=qemu.git;a=blob;f=docs/interop/firmware.json;h=240f565397ae0e754e85976f2d4b5e3873ae8211;hb=HEAD#l311
+	vmkdir usr/share/qemu
+	mv ${DESTDIR}/usr/lib/qemu/firmware ${DESTDIR}/usr/share/qemu/
```

Comment:
If the ultimate destination is `usr/share/qemu` cant `--datadir=/usr/lib` be removed or replaced with `--datadir=/usr/share` when calling `configure`?

Reply:
No,
Right now, the firmware is installed into `/usr/lib/qemu`, i.e. `/usr/lib/qemu/multiboot.bin` those files are sources of warnings in pkg-lint-elf-in-usr-share.

The json files in `/usr/lib/qemu/firmware` must be moved, let's see what Hoshpak propose.